browser/resources/settings/brave_privacy_page/brave_personalization_options.js

```diff
@@ -46,6 +47,9 @@ Polymer({
     this.browserProxy_.getP3AEnabled().then(enabled => {
       this.p3aEnabled_ = enabled;
     });
+    this.browserProxy_.getStatsUsagePingEnabled().then(enabled => {
```

Comment:
I think we also need to add listener like p3a has `p3a-enabled-changed` event to update all other already opened settings page.

Reply:
fixed